Sentry.xcodeproj/project.pbxproj

```diff
@@ -1803,6 +1803,8 @@
 				7BD729972463E93500EA3610 /* SentryDateUtil.m */,
 				7B98D7E325FB7A7200C5A389 /* SentryAppState.h */,
 				7B98D7E725FB7BCD00C5A389 /* SentryAppState.m */,
+				7B8ECBF926498906005FE2EF /* SentryAppStateManager.h */,
+				7B8ECBFB26498958005FE2EF /* SentryAppStateManager.m */,
```

Comment:
I moved these files from the AppStartTracking folder to the Helpers folder. It was annoying to search for them inside of the AppStartTracking integration when working on the OOM integration.